src/components/navigation-header/components/dropdown-menu/index.tsx

```diff
@@ -12,7 +12,7 @@ import deriveKeyEventState from 'lib/derive-key-event-state'
 import ProductIcon from 'components/product-icon'
 import Text from 'components/text'
 import {
-  NewNavigationHeaderItem as NavigationHeaderItem,
+  NavigationHeaderItem,
```

Comment:
I removed the old `NavigationHeaderItem` in this PR, so I was able to rename `NewNavigationHeaderItem` to `NavigationHeaderItem`.